content/docs/examples/advanced-egress/egress-gateway-mtls-origination/index.md

```diff
@@ -0,0 +1,533 @@
+---
+title: Mutual TLS origination by Egress Gateway
+description: Describes how to configure an Egress Gateway to perform mutual TLS origination to external services.
+weight: 45
+keywords: [traffic-management,egress]
+---
+
+The [Configure an Egress Gateway](/docs/examples/advanced-egress/egress-gateway) example describes how to configure
+Istio to direct the egress traffic through a dedicated service called _Egress Gateway_. This examples shows how to
+configure an Egress Gateway to perform mutual TLS origination to external services.
+You will deploy an [NGINX](https://www.nginx.com/) server in your Kubernetes cluster without injecting Istio sidecar
```

Comment:
@rcaballeromx whats the language format we need to use? `We` or `You` ?

Reply:
Always "you"

Reply:
@rshriram We use the second person singular: you, your. However, we do not use the future tense. Thus, line 11 should read:
You deploy ...